### ISSUE TITLE
Add toggle to DevHud for predictions warning

### ISSUE THREAD

**Issue by jlangevin** [codeforboston/cliff-effects#848]
Add 'off' switch to dev hud.  "Maybe we can put an 'off' switch for it in DevSwitch temporarily?"

**Comment by knod** [comment]
Yeah, maybe something that can work even when the dev hud is off so that we can do presentations with the app both without the warning showing and without the dev hud showing.

**Comment by jlangevin** [comment]
Yup, since there's no admin for the app, the toggle will like in DevHud, but the flag will not be tied to DevHud on/off state, it'll live in app state so it's persisted whether DevHud is on or off.
